src/store/search.js

```diff
@@ -22,6 +22,10 @@ export const defaultFacetNames = [
 const filtersFromQf = (qfs) => {
   const filters = {};
 
+  if (!qfs || !qfs[0]) {
```

Comment:
Why is this needed?

Reply:
It breaks on organisation pages without a check.
qfs could be null on these pages.

Reply:
I have pushed a change to address (https://github.com/europeana/portal.js/pull/1125/commits/4e640813aa1d1c08c7f775a0cc59fb1504e701bc) this in a different way, returning the fielded filter needed (I think) for the organization URIs.